src/apicompat/baselines/ApiCompatBaseline.netstandard1.6.txt

```diff
@@ -1,3 +1,10 @@
+Compat issues with assembly System.ComponentModel.TypeConverter:
+CannotSealType : Type 'System.ComponentModel.BaseNumberConverter' is effectively (has a private constructor) sealed in the implementation but not sealed in the contract.
```

Comment:
@ericstj @wtgodbe any idea where these are coming from? Those changes are new after I rebased to master. Did Arcade change?

Reply:
@Anipik added better checks to catch the exact case that resulted in this bug.

Reply:
What does this baseline file indicate BTW? netstandard 1.6 didn't have BaseNumberConverter.

Reply:
Yes it did: https://github.com/dotnet/corefx/blob/82cce14f0c8188a2218ec5dc9a55977ccdba02e0/src/System.ComponentModel.TypeConverter/ref/project.json#L11

Reply:
@wtgodbe I don't understand exactly why this wouldn't have failed CI.  Maybe you're auto-baselining.  That's fine to do for local builds, but maybe you should turn it off on CI.

Reply:
Gotcha, so these changes are expected and we just didn't baseline those before.

Reply:
@ericstj good idea - this should be a quick fix, I'll whip something up now.

Reply:
Might not have time today, I'll circle back tomorrow morning - https://github.com/dotnet/standard/issues/1184